elastic/kibana
# [ResponseOps][Alerting] Index threshold alert can't use unsigned long data type

Resolves [#81786](https://github.com/elastic/kibana/issues/81786)

## Summary

Added `unsigned_long` data types to the index threshold rules.


### Checklist



### To verify
- Create an index with `unsigned_long` data
```
PUT time_index
{
  "mappings": {
    "properties": {
      "my_counter": {
        "type": "unsigned_long"
      },
      "@timestamp": {
        "type": "date"
      }
    }
  }
}

POST /time_index/_bulk?refresh
{"index":{"_id":1}}
{"@timestamp": "2020-10-19T01:01:01.000", "my_counter": 0, "string_counter": "0"}
{"index":{"_id":2}}
{"@timestamp": "2020-10-19T01:01:02.000", "my_counter": 9223372036854775808, "string_counter": "9223372036854775808"}
{"index":{"_id":3}}
{"@timestamp": "2020-10-19T01:01:03.000", "my_counter": 18446744073709551614, "string_counter": "18446744073709551614"}
{"index":{"_id":4}}
{"@timestamp": "2020-10-19T01:01:04.000", "my_counter": 18446744073709551615, "string_counter": "18446744073709551615"}
```
- Create an index threshold rule and verify that you can select `my_counter`
